### DISCUSSION TITLE
Zero Template Custo CSS problems please cancel automatic calls to bootstrap

### DISCUSSION THREAD

**Discussion by LogoJorge** [templaza/astroid-framework#396] [Ideas]
Hello, I have used astroid for a few years, because it is the best template so far that I have used for joomla.

But I must say, however, that my custom CSS in the custom codes section brings me a lot of design problems, my custom design does not pay attention to me because there are some calls to bootstrap that override my designs.

I want a button or option where you can choose Bootstrap Version or choose no bootstrap.

I know that many people use it but it is harming me. Thank you

**Comment by nidzo80** [comment]
@LogoJorge 
![2023-10-25_214508](https://github.com/templaza/astroid-framework/assets/21330927/08d99e9c-fb6b-4487-ac1a-ea80ee189708)

**Comment by sonvnn** [comment]
In fact, Astroid and Joomla are based on Bootstrap 5.3.2. So I recommend you change your styles to suit Bootstrap. Because if you remove it then you will have more problems.
